checks-superstaq/checks_superstaq/pytest_.py

```diff
@@ -86,7 +86,8 @@ def run(
         integration_setup()
 
     return subprocess.call(
-        ["python", "-m", "pytest", *files, *args_to_pass], cwd=check_utils.root_dir
+        ["python", "-m", "pytest", *files, *args_to_pass, "-n=auto"],
```

Comment:
can we add `"-n=auto"` to the addopts line in pyproject.toml instead of passing it here?